packages/theme-check-common/src/checks/unused-assign/index.spec.ts

```diff
@@ -85,7 +85,7 @@ describe('Module: UnusedAssign', () => {
   });
 
   it('should not report unused assigns for things used in a raw-like tag', async () => {
-    const tags = ['style', 'javascript', 'stylesheet'];
+    const tags = ['style'];
```

Comment:
that was wrong. 

> Liquid isn't rendered in {% javascript %} or {% stylesheet %} tags. Including Liquid code in these tags can cause syntax errors, or prevent styles from being applied to the theme.

[source](https://shopify.dev/docs/storefronts/themes/best-practices/javascript-and-stylesheet-tags#stylesheet:~:text=Liquid%20isn%27t%20rendered%20in%20%7B%25%20javascript%20%25%7D%20or%20%7B%25%20stylesheet%20%25%7D%20tags.%20Including%20Liquid%20code%20in%20these%20tags%20can%20cause%20syntax%20errors%2C%20or%20prevent%20styles%20from%20being%20applied%20to%20the%20theme.)